addon/utils/plugin-editor-api.js

```diff
@@ -18,6 +20,17 @@ export default class PluginEditorApi {
     this._hrId = hrId;
   }
 
+  executeCommand(name, ...args) {
```

Comment:
already added these so plugins can use the virtual dom in the future, not used anywhere at the moment